tests/pytests/conftest.py

```diff
@@ -95,6 +98,21 @@ def get_image_label(image, label_type='pds3'):
 
     return label_file[0]
 
+def get_table_data(image, table_name):
```

Comment:
This function is designed to be used with the output of isis's blobdump app which dumps a  BLOB/Table to a separate file with the Table's PVL label and then the binary data.